examples/generic_connector_configs/README.md

```diff
@@ -217,6 +218,40 @@ to the backend server uses SSL.
 
 ___
 
+### Stripe API
+This example can be used to interact with [Stripe's API](https://stripe.com/docs/api).
+
+The configuration file for the Splunk Web API can be found at [stripe_secretless.yml](./stripe_secretless.yml).
+
+#### How to use this connector
+* Get the [Stripe API Key](https://dashboard.stripe.com/apikeys), which can be used as a Bearer token
+* Get an [connected account](https://stripe.com/docs/connect/authentication) or generate an [idempotency key](https://stripe.com/docs/api/idempotent_requests) if needed
+* Query the Striple API using `http_proxy=localhost:80*1 curl api.slack.com/{route}`.
+This example supports several header configurations, so it is recommended to
+look at [stripe_secretless.yml](./stripe_secretless.yml) to figure out which
```

Comment:
Maybe put one sentence comments in the yaml file to specify what each one does. Do you have to delete the other ones you aren't going to use first?